ale/drivers/mro_drivers.py

```diff
@@ -147,4 +147,16 @@ def spacecraft_name(self):
         name_lookup = {
             'MARS_RECONNAISSANCE_ORBITER': 'MRO'
         }
-        return name_lookup[self.label['SPACECRAFT_NAME']]
+        return name_lookup[super().spacecraft_name]
+
+    @property
+    def detector_start_line(self):
+        return 0
```

Comment:
I think...I didn't see this in the IK or the label.

Reply:
This is an image specific thing. Should only show up in the label.

Reply:
:+1: I am not seeing this in any CTX labels, but I am viewing a super tiny sample. Do we have an example where it does so that I can get the correct label keyword? Or I'll get this from the SIS (see my other comment below).

Reply:
I don't think there are any CTX images where this isn't 1. The default to start at the top of the detector is 1.